src/presentation/controller/registration/Routing.ts

```diff
@@ -58,19 +58,26 @@ const registrationRoutingGeneralPartners = {
 const registrationRoutingGeneralPartnerChoice = {
   previousUrl: url.GENERAL_PARTNERS_URL,
   currentUrl: url.GENERAL_PARTNER_CHOICE_URL,
-  nextUrl: url.ADD_GENERAL_PARTNER_PERSON_URL,
+  nextUrl: "",
   pageType: RegistrationPageType.generalPartnerChoice
 };
 
-const registrationRoutingAddGeneralPartner = {
+const registrationRoutingAddGeneralPartnerPerson = {
   previousUrl: url.GENERAL_PARTNER_CHOICE_URL,
   currentUrl: url.ADD_GENERAL_PARTNER_PERSON_URL,
   nextUrl: url.LIMITED_PARTNERS_URL,
   pageType: RegistrationPageType.addGeneralPartnerPerson
 };
 
+const registrationRoutingAddGeneralPartnerLegalEntity = {
+  previousUrl: url.GENERAL_PARTNER_CHOICE_URL,
+  currentUrl: url.ADD_GENERAL_PARTNER_LEGAL_ENTITY_URL,
+  nextUrl: url.LIMITED_PARTNERS_URL,
```

Comment:
You need to get AC4 of the story amended to reflect where this page will go to. I had to do this also for AC4 of my 'General Partner Person' story.

Reply:
updated the story now.